content/en/docs/contribute/style/style-guide.md

```diff
@@ -38,6 +38,45 @@ The English-language documentation uses U.S. English spelling and grammar.
 
 {{< comment >}}[If you're localizing this page, you can omit the point about US English.]{{< /comment >}}
 
+## Guidelines for contributing content
+
+Kubernetes documentation only hosts content from projects within the
+ `kubernetes` or `kubernetes-sigs` GitHub organizations or from current CNCF
+  projects ([Graduated/Incubating](https://www.cncf.io/projects/), [Sandbox](https://www.cncf.io/sandbox-projects/)).
+The CNCF doesn't maintain a list of archived projects. However, you can
+ search the CNCF Technical Oversight Committee GitHub 
+ [pull requests](https://github.com/cncf/toc/labels/archive) to find
+  proposals that are for archiving projects.
+
+Kubernetes documentation does not host duplicate content sourced from multiple 
```

Comment:
s/host/include/ and I think this paragraph is clear.

Reply:
removed the word 'host'